Configurations.md

```diff
@@ -906,6 +906,37 @@ fn lorem() -> usize {
 
 See also [`control_brace_style`](#control_brace_style).
 
+
+## `where_single_line`
+
+To force single line where layout
+
+- **Default value**: `false`
+- **Possible values**: `true`, `false`
+
+#### `false` (default):
+
+```rust
+impl<T> Lorem for T
+where
+    Option<T>: Ipsum,
+{
+    ...
+}
+```
+
+#### `true`:
+
+```rust
+impl<T> Lorem for T where
```

Comment:
The `where` needs to be on the next line. cc [the style guide](https://github.com/rust-lang-nursery/fmt-rfcs/blob/master/guide/items.md#option---where_single_line).